extensions/amp-date-countdown/1.0/amp-date-countdown.js

```diff
@@ -14,19 +14,17 @@
  * limitations under the License.
  */
 
-import {DateCountdown} from './date-countdown';
-import {PreactBaseElement} from '../../../src/preact/base-element';
+import {BaseElement} from './base-element';
 import {Services} from '../../../src/services';
 import {dev, userAssert} from '../../../src/log';
 import {dict} from '../../../src/utils/object';
 import {isExperimentOn} from '../../../src/experiments';
 import {isLayoutSizeDefined} from '../../../src/layout';
-import {parseDateAttrs as parseDateAttrsBase} from '../../../src/utils/date';
 
 /** @const {string} */
 const TAG = 'amp-date-countdown';
 
-class AmpDateCountdown extends PreactBaseElement {
+class AmpDateCountdown extends BaseElement {
```

Comment:
I might need to implement `isLayoutSupported` in `BaseElement`; not 100% sure but I think that's part of the Bento side (`userAssert`/experiment checking is not ofc)

Reply:
Done; please verify

Reply:
Removed per offline convo